src/noit_check_rest.c

```diff
@@ -396,7 +396,7 @@ rest_show_check(mtev_http_rest_closure_t *restc,
   xmlNodePtr node, root, attr, config, state, tmp, anode;
   uuid_t checkid;
   noit_check_t *check;
-  char xpath[1024], *uuid_conf, *module = NULL, *value = NULL;
+  char xpath[1024], *uuid_conf = NULL, *module = NULL, *value = NULL;
```

Comment:
I guess you mean pobj instead of doc (see last commit)
doc is not initialized in line 425 yet